static/app/views/performance/utils/index.tsx

```diff
@@ -369,11 +369,7 @@ export function getProject(
   eventData: EventData,
   projects: Project[]
 ): Project | undefined {
-  const projectSlug = (eventData?.project as string) || undefined;
-
-  if (typeof projectSlug === undefined) {
```

Comment:
`typeof` returns a string, it can never equal `undefined`... but it could equal `"undefined"`

Reply:
both `no-constant-binary-expression` and `valid-typeof` would catch this statement.